packages/discovery-react-components/src/utils/document/processDoc.ts

```diff
@@ -322,8 +315,16 @@ function setupSectionParser(
           parser.popState();
         }
       }
+      if (tagName === 'body') {
```

Comment:
Add a empty line above this. Breaking up the code can make it easier to visually parse/read.